### ISSUE TITLE
 v0.9.7 NPE at startup

### ISSUE THREAD

**Issue by matosf** [zalando-stups/spring-boot-zalando-stups-tokens#2]
Hi,

After updating from v0.9.6 to v0.9.7 I now get the following NPE:

```
Caused by: java.lang.NullPointerException (i.e. beanFactory is null):
    at org.zalando.stups.tokens.AccessTokensBean.configureScheduler(AccessTokensBean.java:215) ~[spring-boot-zalando-stups-tokens-0.9.7.jar:?]
    at org.zalando.stups.tokens.AccessTokensBean.start(AccessTokensBean.java:152) ~[spring-boot-zalando-stups-tokens-0.9.7.jar:?]
```

Quick fix for now is to add the following configuration:

```
tokens:
  use-existing-scheduler: false
```

Please have a look.

**Comment by jbellmann** [comment]
thanks for reporting will look into this

**Comment by jbellmann** [comment]
Do you have somewhere in your application code like this

```
@Bean
public AccessTokensBean accessTokensBean(){
    return new AccessTokensBean(...properties ....);
}

or in short, do initialise the bean 'by hand' ?

or do you have 'startAfterCreation'-property set to 'true'?
```

**Comment by matosf** [comment]
I have the startAfterCreation enabled.
Em 10/03/2016 1:53 da tarde, "Jörg Bellmann" notifications@github.com
escreveu:

**Comment by jbellmann** [comment]
0.9.10 released some minutes ago should solve this.

If it's working, please close the issue.

**Comment by matosf** [comment]
confirmed that is fixed with 0.9.10. thanks!
